### DISCUSSION TITLE
Tasmota Zigbee Bridge: Ikea Styrbar W2349 not working

### DISCUSSION THREAD

**Discussion by evaluationevaluation** [arendst/Tasmota#21230] [Support request] [answered]
Hi Tasmota-Team and Supporters,

first of all I want to really thank you very, very much for your engagement! I use Tasmota for years now and am really happy with it!!
So far most of the time I was able to solve issues by reading & trial/error but this time I fail.

Short version:
I use several Tradfri and Styrbar remotes from IKEA for years now and just bought a new Styrbar.
With ZBPermitJoin I included the device into the ZB net and with ZBStatus shows me that the new device (0xB882) has been accepted.

`
15:02:32.506 MQT: stat/TasmotaZigbeeBridge/RESULT = {"ZbStatus1":[{"Device":"0x0769","Name":"TradfriRemote1"},{"Device":"0x3B28","Name":"TradfriRemote2"},{"Device":"0xE467","Name":"TradfriBulb1000"},{"Device":"0xD893","Name":"TradfriRemote3"},{"Device":"0x47BD","Name":"TradfriRemote4"},{"Device":"0x294D","Name":"TradfriRemote5"},{"Device":"0x76D7","Name":"TradfriRemote6"},{"Device":"0x1810","Name":"WindowSensor1"},{"Device":"0x9850","Name":"WindowSensor2"},{"Device":"0x4DFF","Name":"WindowSensor3"},{"Device":"0x5264","Name":"WindowSensor4"},{"Device":"0x429C","Name":"WindowSensor5"},{"Device":"0xB882","Name":"TradfriRemote7"}]}
`

But I don't see any button events in commands. All other devices remotes and window sensors are running fine.

Longer version:
I have Sonoff Tasmota ZigbeeBridge (Model: ZBBridge) with Tasmota Release 9.3.1 in use - as it works well for a long time I did not update. I remember that flashing the device was not that easy and I feared to get issues...
ZigbeeBridge events are being transferred via MQTT to NodeRed (if this is of interest).
After pairing I see the new device in the WebUI with 0 minutes. But then no button press can stop/reset it.
All other Ikea remotes are working fine.
TasmotaRemote6 (0x76D7) is model Styrbar W2119.
The new device TasmotaRemote7 (0xB882) is model Styrbar W2349.
Here are some of my tests to compare TasmotaRemote 6 and 7.

`
15:11:15.678 MQT: tele/TasmotaZigbeeBridge/SENSOR = {"ZbInfo":{"0xB882":{"Device":"0xB882","Name":"TradfriRemote7","IEEEAddr":"0x3C2EF5FFFEEE3816","ModelId":"Remote Control N2","Manufacturer":"IKEA of Sweden","Endpoints":[1],"Config":[],"Reachable":false,"LastSeen":1384,"LastSeenEpoch":1713620891,"LinkQuality":128}}}

15:11:49.350 MQT: tele/TasmotaZigbeeBridge/SENSOR = {"ZbInfo":{"0x76D7":{"Device":"0x76D7","Name":"TradfriRemote6","IEEEAddr":"0x842E14FFFE5DBFE0","ModelId":"Remote Control N2","Manufacturer":"IKEA of Sweden","Endpoints":[1],"Config":[],"Reachable":true,"LastSeen":154,"LastSeenEpoch":1713622155,"LinkQuality":29}}}

15:12:59.334 MQT: tele/TasmotaZigbeeBridge/SENSOR = {"ZbReceived":{"0xB882":{"Device":"0xB882","Name":"TradfriRemote7","Reachable":false}}}
15:13:05.026 ZIG: {"ZbEZSPReceived":"80004282B8"}
15:13:05.030 MQT: tele/TasmotaZigbeeBridge/RESULT = {"ZbRouteError":{"ShortAddr":"0xB882","Status":66,"StatusMessage":"MAC_INDIRECT_TIMEOUT"}}
15:13:14.282 ZIG: {"ZbEZSPReceived":"80004282B8"}
15:13:14.289 MQT: tele/TasmotaZigbeeBridge/RESULT = {"ZbRouteError":{"ShortAddr":"0xB882","Status":66,"StatusMessage":"MAC_INDIRECT_TIMEOUT"}}
15:13:23.536 ZIG: {"ZbEZSPReceived":"80004282B8"}
15:13:23.543 MQT: tele/TasmotaZigbeeBridge/RESULT = {"ZbRouteError":{"ShortAddr":"0xB882","Status":66,"StatusMessage":"MAC_INDIRECT_TIMEOUT"}}

15:12:22.439 MQT: stat/TasmotaZigbeeBridge/RESULT = {"ZbPing":"Done"}
15:12:24.467 MQT: tele/TasmotaZigbeeBridge/SENSOR = {"ZbReceived":{"0x76D7":{"Device":"0x76D7","Name":"TradfriRemote6","Reachable":false}}}
15:12:30.163 ZIG: {"ZbEZSPReceived":"800042D776"}
15:12:30.168 MQT: tele/TasmotaZigbeeBridge/RESULT = {"ZbRouteError":{"ShortAddr":"0x76D7","Status":66,"StatusMessage":"MAC_INDIRECT_TIMEOUT"}}
15:12:39.418 ZIG: {"ZbEZSPReceived":"800042D776"}
15:12:39.422 MQT: tele/TasmotaZigbeeBridge/RESULT = {"ZbRouteError":{"ShortAddr":"0x76D7","Status":66,"StatusMessage":"MAC_INDIRECT_TIMEOUT"}}
15:12:48.671 ZIG: {"ZbEZSPReceived":"800042D776"}
15:12:48.675 MQT: tele/TasmotaZigbeeBridge/RESULT = {"ZbRouteError":{"ShortAddr":"0x76D7","Status":66,"StatusMessage":"MAC_INDIRECT_TIMEOUT"}}
`

Can anyone please give he a hint what I can check/do?
Is this perhaps a known issue and solved in a newer version of TasmotaZigbeeRelease?
Is it easy and without big risk to upgrade to the latest version of TasmotaZigbeeBridge?

Thanks a lot in advance for your advise!
 Eva

**Comment by s-hadinger** [comment]
9.3 is really old, and I don't remember what features were present or not. However upgrading the coordinator has a high chance of losing Zigbee configuration.

There were many improvements to automatically to bindings at pairing time, this may be now required for Ikea devices with newer firmwares.

**Comment by evaluationevaluation** [comment, marked as answer]
found the solution in the meanwhile...

I replaced the item by a new one and get a similar but not identical issue with it.
Some buttons were working and some not.

Then I found the autobinding feature and saw that I actively can bind things by myself.
I just expected some binding numbers to be consecutive and tried and finally all buttons are working fine.

ZigbeeBridge is still v9.3.1.

Perhaps this helps someone with similar issues.
I really love to use those IKEA multi-button devices, especially the Styrbar version, for remote control!
